src/components/Navigation.jsx

```diff
@@ -1,5 +1,70 @@
+"use client";
+import React, { useState } from "react";
+import Navbar from "react-bootstrap/Navbar";
+import Link from "next/link";
+import Nav from "react-bootstrap/Nav";
+import { items } from "@/data/navigation";
+import Image from "next/image";
+import logo from "@/public/prishi-text-logo.webp";
+
 const Navigation = () => {
-  return <div>Navigation</div>;
+  const [selected, setSelected] = useState("");
+  return (
+    <Navbar
+      collapseOnSelect
+      expand="md"
+      fixed="top"
+      className="bg-rishi-white p-4"
+    >
+      <Navbar.Brand>
+        <Link
+          onClick={() => setSelected("")}
```

Comment:
get rid of this whole line, this is preventing the orange circle from showing up